python/ray/train/_internal/state/schema.py

```diff
@@ -47,6 +47,55 @@ class TrainWorkerInfo(BaseModel):
     )
 
 
+@DeveloperAPI
+class MemoryInfo(BaseModel):
+    rss: int
+    vms: int
+    pfaults: Optional[int]
+    pageins: Optional[int]
+
+
+@DeveloperAPI
+class ProcessStats(BaseModel):
```

Comment:
Is this schema based on an existing one? See questions below

Reply:
yeah, this is the existing gpus schema for nodes and actors. I agree it's ugly...

I think with export API, we have a chance to really clean this up for the future but let's not bundle it in with the train dashboard changes.